include/taskolib/Sequence.h

```diff
@@ -540,18 +566,16 @@ class Sequence
     /// Empty if indentation is correct and complete, error message otherwise
     std::string indentation_error_;
 
-    /// Sequence label.
-    std::string label_;
+    std::string label_; ///< Sequence label.
 
-    /// Step setup script.
-    std::string step_setup_script_;
+    std::string step_setup_script_; ///< Step setup script.
 
-    /// Collection of steps.
-    std::vector<Step> steps_;
+    std::vector<Step> steps_; ///< Collection of steps.
 
-    /// Flag to determine if the sequence is running.
-    bool is_running_{false};
+    bool is_running_{false}; ///< Flag to determine if the sequence is running.
 
+    Timeout timeout_{Timeout::infinity()}; ///< Timeout for sequence.
+    TimeoutTrigger timeout_trigger_;
```

Comment:
If we want to keep the `TimeoutTrigger` class, I think we should remove the `timeout_` member. It can just as well be stored inside `timeout_trigger_`. This way we just store redundant information and make it unclear which one is valid.

Reply:
Yup! I was also thinking about removing it. Fixed.